python/paddle/distributed/passes/auto_parallel_replace_with_parallel_cross_entropy.py

```diff
@@ -73,6 +73,12 @@ def _apply_single_impl(self, main_program, startup_program, context):
                     if not placement1[1].is_shard():
                         return
 
+                    process_ids = op.dist_attr.process_mesh.process_ids
```

Comment:
Why use process_ids from `op.dist_attr.process_mesh` instead of  process_ids from 'operand1'?

Reply:
My understanding of this is in paddle computation graph's value is an edge and op is a node, I have considered that the process_ids in op and operand might be different as well. So I guess the process_ids obtained from op may be suitable here.